style.css

```diff
@@ -100,6 +102,282 @@ ul li {
   padding: 0;
 }
 
+/* wrapper section end */
+
+/* recent works section start */
+
+.recent-works {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  padding-left: 60px;
+  padding-right: 60px;
+}
+
+.works-title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 80px;
+}
+
+.recent-works .works-title h2 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+  text-align: center;
+}
+
+.solid-line {
+  border: 1px solid #bbb;
+  width: 100%;
+  margin-top: 60px;
+  margin-bottom: 60px;
+}
+
+/* recent works section end */
+
+/* all projects multi section start */
+
+.all-projects {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  padding: 50px;
+  width: 100%;
+  padding-bottom: 70px;
+}
```

Comment:
- [ ] As stated the project requirement In order to lay out the elements in "Works" section you should use **CSS Grid and Flexbox** combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card. But I see that you used in the reverse way Kindly Fix it.